CHANGELOG.md

```diff
@@ -0,0 +1,91 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [0.3.0] - 2020-03-30
+
+### Added
+
+- Added `CHANGELOG.md`
+- Added `$_ZO_EXCLUDE_DIRS` to prevent certain directories from being added to the database.
+- Added support for generic POSIX shells.
+
+### Changed
+
+- Switched default database location to user's data directory.
```

Comment:
I think it looks nicer (when viewing it as raw text) to have the header be directly above what it is describing. If you're fine with it, I'd like to ask that all headers follow this same kind of decoration: Header, directly followed by entries, with a newline separating the next header.

```suggestion
### Changed
- Switched default database location to user's data directory.
```

Reply:
I'm not sure about this - [`prettier`](https://prettier.io/), which is the Markdown formatter I use, adds in blanks before and after headers, and [`markdownlint`](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md022---headings-should-be-surrounded-by-blank-lines) also seems to agree.

Reply:
Fair enough. Even my "gold standard of changelogs" Alacritty agrees!